files/ui/advancedpage.ui

```diff
@@ -14,7 +14,7 @@
    <item>
     <widget class="QTabWidget" name="AdvancedTabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
This looks like a redundant change. It makes second "Visuals" tab default instead of "Game Mechanics".